astropy/config/tests/test_configs.py

```diff
@@ -17,56 +17,59 @@
 
 
 def test_paths():
-    assert 'astropy' in paths.get_config_dir()
-    assert 'astropy' in paths.get_cache_dir()
+    assert 'astropy' in paths.get_config_dir('astropy')
+    assert 'astropy' in paths.get_cache_dir('astropy')
```

Comment:
Related to the above: this test has to be left unchanged to not be a public API-breaking change.  Same with all the stuff below in this file.